mapviz_plugins/mapviz_plugins.xml

```diff
@@ -25,9 +25,9 @@
   <class name="mapviz_plugins/textured_marker" type="mapviz_plugins::TexturedMarkerPlugin" base_class_type="mapviz::MapvizPlugin">
     <description>Textured marker mapviz plugin.</description>
   </class>
-  <class name="mapviz_plugins/move_base" type="mapviz_plugins::MoveBasePlugin" base_class_type="mapviz::MapvizPlugin">
+  <!--class name="mapviz_plugins/move_base" type="mapviz_plugins::MoveBasePlugin" base_class_type="mapviz::MapvizPlugin">
```

Comment:
Well, it shouldn't be uncommented since we didn't port the MoveBase plugin, but it'd probably be good to add a comment explaning why it's commented out, so I'll do that.